conf/server.properties

```diff
@@ -0,0 +1,50 @@
+#Minecraft server properties
+#Sat Sep 11 17:53:29 GMT 2021
+enable-jmx-monitoring=false
+rcon.port=25575
+gamemode=survival
+enable-command-block=false
+enable-query=false
```

Comment:
What is this?

Reply:
from minecraft.fandom.com
> Enables GameSpy4 protocol server listener. Used to get information about server. 
I think we should just keep it as false